### PR TITLE
explorer: link network select to appStore

### LINKED ISSUE #1731: explorere: Link the network select dropdown to the appStore
#### Summary
Link the network select dropdown options to the applications store.

### PR DESCRIPTION
**Pull request by deuch13**
Resolves #1731

### CODE REVIEW

File: explorer/src/lib/components/navbar/Navbar.svelte [hunk 1]
```diff
@@ -56,11 +46,21 @@
     )[0]?.clientHeight;
   }
 
+  /**
+   * @param {Event} e
+   */
+  function handleChange(e) {
```
Comment (ascartabelli) [inline explorer/src/lib/components/navbar/Navbar.svelte:52]: I think `handleNetworkChange` would be a more fitting name, as we have many components here.

File: explorer/src/lib/containers/statistics-panel/StatisticsPanel.svelte [hunk 2]
```diff
@@ -43,7 +43,9 @@
 
   onNetworkChange((network) => {
     marketDataStore.getData(network);
+    nodeLocationsStore.reset();
```
Comment (ascartabelli) [inline explorer/src/lib/containers/statistics-panel/StatisticsPanel.svelte:46]: Is a reset really needed here? If yes why on the node locations but not on the market data?

**Review by ascartabelli — COMMENTED** [review]
A couple of minor notes, good to merge otherwise

File: explorer/src/lib/containers/statistics-panel/StatisticsPanel.svelte (hunk 2 above)
Comment (deuch13) [reply explorer/src/lib/containers/statistics-panel/StatisticsPanel.svelte:46]: Yes, it is needed. I do not reset the market data as that api endpoint is not dependent on the node.

**Review by ascartabelli — APPROVED** [review]

**Pull request merged**